parser/metadata.go

```diff
@@ -21,132 +21,96 @@ import (
 
 var (
 	SystemLocalColumns = []*message.ColumnMetadata{
```

Comment:
Do we need to worry about any of the special DSE columns?

Reply:
`cql-proxy` is pretending to be C*. Most (All?) drivers will connect to C* without expecting those columns and only a subset of datastax driver will use them.